enginetest/queries/query_plans.go

```diff
@@ -11337,8 +11229,8 @@ WHERE
 			" │       │           │   ├─ aac.id:74!null\n" +
 			" │       │           │   └─ sl3s5.M22QN:63!null\n" +
 			" │       │           └─ TableAlias(aac)\n" +
-			" │       │               └─ IndexedTableAccess(TPXBU)\n" +
-			" │       │                   ├─ index: [TPXBU.id]\n" +
+			" │       │               └─ Table\n" +
+			" │       │                   ├─ name: TPXBU\n" +
```

Comment:
Is removing `IndexedTableAccess` here a problem?

Reply:
Oh good catch, this is a weird one. If we successfully push a filter through a subquery, `applyIndexesFromOuterScope` is missing the outer filter for indexing. Logic that depends on pushdown failing sporadically.